multinet.rst

```diff
@@ -1,12 +1,122 @@
-.. _multinet:
+Using Multinet
+==============
 
-========
-Multinet
-========
-The Multinet API and web application
+This chapter is all about how to make use of a running Multinet system. Along
+the way, you will also learn how Multinet structures data into tables and
+networks, collected in different workspaces, and how Multinet thinks of
+multivariate data and multivariate networks.
 
-.. toctree::
-   :maxdepth: 2
+Getting Started
+---------------
 
-   installation
-   testing
+To begin, you will need access to a running Multinet instance (either the
+official instance hosted at https://multinet.app, or a local instance you have
+built and run; see :ref:`quickstart` for help). We will follow some steps to
+illustrate the basic mode of using the system.
+
+**1. Create a Workspace**
+
+Multinet structures all the data it hosts into *workspaces*. A workspace can
+hold several tables of data, as well as several networks assembled from those
+tables. Think of a workspace as a dedicated area for related data to be stored.
+
+In this tutorial sequence we will be recreating the `Paul Revere dataset
+<https://kieranhealy.org/blog/archives/2013/06/09/using-metadata-to-find-paul-revere/>`_
+as a social network connecting the various Revolutionary War figures to the
+social clubs they belonged to.
+
+To create a workspace to hold the materials for this dataset, use the ``NEW
+WORKSPACE`` button appearing near the top of the left sidebar. A dialog box will
+appear asking for the name of the new workspace. Fill this in with ``boston``
+and click on the ``CREATE WORKSPACE`` button. This will create the workspace and
+activate it.
```

Comment:
```suggestion
activate it. You can tell it's active by the shading in the list on the left side of the screen.
```

Reply:
I think this bit will be self-evident.

Reply:
Although, eventually it might be a good idea to include screencaps for the major steps, in which case some of the existing language could be cut away too.

Reply:
I was just thinking that this might help us reduce some confusion down the road. Up to you